.changeset/few-maps-smell.md

```diff
@@ -0,0 +1,74 @@
+---
+'@commercetools-uikit/design-system': minor
```

Comment:
I marked only changes in 'visual-testing-app' and '@commercetools-uikit/design-system' as minor, perhaps all changes should be marked as minor?